From 53fc92d88ebbbfcf2cfd67626fe8cd19175aa5a6 Mon Sep 17 00:00:00 2001
From: Hanna Hawa <hannah@marvell.com>
Date: Mon, 9 Oct 2017 15:49:00 +0300
Subject: [PATCH 2029/2241] irqchip/gicv3-its: update cache flushing print
 location

Update cache flushing print location, depends
on ITS_FLAGS_CMDQ_NEEDS_FLUSHING

Change-Id: I52af234e67132b7837e738d6a28675a3c0de094a
Signed-off-by: Hanna Hawa <hannah@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/45081
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Omri Itach <omrii@marvell.com>
---
 drivers/irqchip/irq-gic-v3-its.c | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/drivers/irqchip/irq-gic-v3-its.c b/drivers/irqchip/irq-gic-v3-its.c
index 8af56e8..aeda148 100644
--- a/drivers/irqchip/irq-gic-v3-its.c
+++ b/drivers/irqchip/irq-gic-v3-its.c
@@ -1570,7 +1570,6 @@ static int its_probe(struct device_node *node, struct irq_domain *parent)
 			baser |= GITS_CBASER_nC;
 			writeq_relaxed(baser, its->base + GITS_CBASER);
 		}
-		pr_info("ITS: using cache flushing for cmd queue\n");
 		its->flags |= ITS_FLAGS_CMDQ_NEEDS_FLUSHING;
 	}
 
@@ -1591,6 +1590,9 @@ static int its_probe(struct device_node *node, struct irq_domain *parent)
 		writeq_relaxed(baser, its->base + GITS_CBASER);
 	}
 
+	if (its->flags & ITS_FLAGS_CMDQ_NEEDS_FLUSHING)
+		pr_info("ITS: using cache flushing for cmd queue\n");
+
 	writeq_relaxed(0, its->base + GITS_CWRITER);
 	writel_relaxed(GITS_CTLR_ENABLE, its->base + GITS_CTLR);
 
-- 
2.7.4

